pyproject.toml

```diff
@@ -1,2 +1,5 @@
 [build-system]
 requires = ["setuptools", "wheel"]
+
+[tool.coverage.run]
```

Comment:
Please move it to a new branch/PR (e.g. `git cherry-pick e6569a1414a66745f10e7f766d887f8bd2d7125c` on a new branch). We need to either create some script `tools/run_coverage.(py/sh)` or add it to CI.